src/hooks/chat/useBodyScrollControllerEffect.tsx

```diff
@@ -44,6 +44,17 @@ export default (
     chatBodyRef.current.scrollTop = scrollTop;
   }, [chats]);
 
+  const lastChat = chats.length > 0 ? chats[chats.length - 1] : {};
```

Comment:
```suggestion
  const lastChat = chats.at(-1);
```

로 더 간단히 할 수 있을 것 같아요.